src/library/include/platform/platform.h

```diff
@@ -71,4 +71,6 @@ static inline void platform_panic(uint32_t p) {}
 
 extern struct timer *platform_timer;
 
+extern struct timesource_data *platform_ts;
```

Comment:
Why do we need this extern structure ?

Reply:
@lgirdwood I need to know the ``clk`` id of platform_timer (eg. CLK_SSP) which is platform dependency.

Reply:
@bardliao ok, can you rebase

Reply:
@lgirdwood Yes, done.